### PR TITLE
LPS-135627 Properly instruct npm-bundler to not bundle the client SDK…

### PR DESCRIPTION
**Pull request by dsanz**
… file

### CODE REVIEW

**Review by nhpatt — APPROVED** [review]

File: modules/apps/remote-app/remote-app-client-js/npmscripts.config.js [hunk 1]
```diff
@@ -13,5 +13,7 @@
  */
 
 module.exports = {
-	bundling: false,
+	build: {
```
Comment (izaera) [inline modules/apps/remote-app/remote-app-client-js/npmscripts.config.js:16]: clap

:sweat_smile:

**Comment by izaera** [comment]
LGTM

Seems it needs a rebase? :thinking:

**Comment by nhpatt** [comment]
Retrying ci:forward because it picked the CI in the middle of the candidate release with a lot of version bumps.

**Comment by nhpatt** [comment]
ci:forward

**Pull request closed without merging**